packages/main/src/queue/index.ts

```diff
@@ -248,36 +248,54 @@ const handlers = {
     }
   }),
 
-  flag: handler({
-    enqueue(params: { accountId: ID; box: { name: string }; uids: number[] }) {
-      cache.addFlag({ ...params, flag: "\\Flagged" })
+  setFlagged: handler({
```

Comment:
Good stuff. Would you remove the `unFlag` handler now that it is not used anymore?